descriptors/ModuleDescriptor-template.json

```diff
@@ -4,11 +4,11 @@
   "provides": [
     {
       "id": "orders",
-      "version": "1.0",
+      "version": "3.0",
       "handlers": [
         {
           "methods": ["POST"],
-          "pathPattern": "/orders",
+          "pathPattern": "/orders/composite-order",
```

Comment:
Please correct to `"pathPattern": "/orders/composite-orders"`

Reply:
fixed